src/Styleguide/Utils/HorizontalPadding.tsx

```diff
@@ -0,0 +1,24 @@
+import { space } from "@artsy/palette"
+import styled, { css } from "styled-components"
+import { ResponsiveSpaceValue } from "styled-system"
+import { media } from "Styleguide/Elements/Grid"
+
+export interface HorizontalPaddingProps {
+  px: ResponsiveSpaceValue
+}
+
+export const HorizontalPadding = styled.div<HorizontalPaddingProps>`
+  ${p =>
+    p.px[1] &&
+    css`
+      padding-right: ${space(p.px[1])}px;
+      padding-left: ${space(p.px[1])}px;
+    `} ${p =>
```

Comment:
If you add a `;` after the ` `}` then the second `${p =>` can sit onto its own line and prettier wont reformat.  

Also, if possible can this go in sequence, like `p.px[0]` and then `px[1]`? It reads a little bit better.